src/main/target/common_fc_pre.h

```diff
@@ -43,13 +43,15 @@
 #define USE_DSHOT
 #define USE_DSHOT_DMAR
 #define USE_GYRO_DATA_ANALYSE
+#define USE_GYRO_FAST_KALMAN
```

Comment:
`USE_GYRO_FAST_KALMAN` should be `#define`d based on ROM size, not processor type.